### PR TITLE
fixed a couple typos

### PR DESCRIPTION
**Pull request by DanielReinagel**
## Description

Fixed a couple of typos that don't affect the code.

Fixes # (issue)

https://trello.com/c/qcwJXk88/156-couple-typos-and-unintentional-mistakes

## Loom Video

Not included (You can just look at the commits and understand it all)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [ ] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by LeonelHays — APPROVED** [review]
nice catch on all three of those. I see no conflicts here

**Review by jonahjtr — APPROVED** [review]
great catch, small and concise pull request. perfect

File: api/admin/adminRouter.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ router.put('/:id', authRequired, checkAdminExist, async (req, res, next) => {
   } catch (error) {
     next({
       status: 400,
-      message: 'soemthing went wrong while updating admin profile',
+      message: 'something went wrong while updating admin profile',
```
Comment (RoseGabriely) [inline api/admin/adminRouter.js:63]: good catch but it looks like this was addressed in a previous open PR

**Review by RoseGabriely — APPROVED** [review]
Great catches thanks for changing these

**Pull request merged**